src/System.Security.Cryptography.X509Certificates/tests/Cert.cs

```diff
@@ -11,12 +13,14 @@ namespace System.Security.Cryptography.X509Certificates.Tests
     //
     internal static class Cert
     {
-        internal const X509KeyStorageFlags EphemeralIfPossible =
+        // netstandard: DefaultKeySet
+        // netcoreapp-OSX: DefaultKeySet
+        // netcoreapp-other: EphemeralKeySet
+        internal static readonly X509KeyStorageFlags EphemeralIfPossible =
 #if netcoreapp11
```

Comment:
The test library builds and runs for both netcore and netstandard.  EphemeralKeySet is a new feature to netcoreapp.